src/layouts/AllServices.test.tsx

```
@@ -0,0 +1,223 @@
+
+      store.set(drawerPanelContentAtom, {
+        scope: 'learningResources',
+        module: './HelpPanel',
+      });
+      expect(store.get(drawerPanelContentAtom)).toBeTruthy();
+
+      store.set(drawerPanelContentAtom, undefined);
+      expect(store.get(drawerPanelContentAtom)).toBeUndefined();
+    });
+  });
+
```

ESLint

[error] Delete `⏎`

(prettier/prettier)